scripts/adminconsole.bat

```diff
@@ -0,0 +1 @@
+java -cp .\lib\admin.jar;.\lib\slf4j-api-1.7.26.jar;.\lib\slf4j-simple-1.6.4.jar;.\lib\jackson-annotations-2.9.0.jar;.\lib\jackson-core-2.9.8.jar;.\lib\jackson-databind-2.9.8.jar;.\lib\jasypt-1.9.2.jar org.apereo.openequella.adminconsole.launcher.ClientLauncher
```

Comment:
We'd be wanting to aim for a fat JAR me thinks. Also, this overlaps with @PenghaiZhang task, but will help with when he does the windows package.